src/Controller/DefaultController.php

```diff
@@ -20,55 +23,47 @@ public function __construct(private DataResponseFactoryInterface $responseFactor
 
     public function get(GeneratorRequest $request): ResponseInterface
     {
-        /** @var AbstractGenerator $generator */
         $generator = $request->getGenerator();
         $params = [
-            'name' => $generator->getName(),
-            'description' => $generator->getDescription(),
-            'template' => $generator->getTemplate(),
-            'templatePath' => $generator->getTemplatePath(),
-            'templates' => $generator->getTemplates(),
+            'name' => $generator::getName(),
+            'description' => $generator::getDescription(),
+            'commandClass' => $generator::getCommandClass(),
+            //            'templatePath' => $generator->getTemplatePath(),
+            //            'templates' => $generator->getTemplates(),
```

Comment:
?

Reply:
I'll fix it later or remove if it is not needed